src/components/activities/manageBillActivity/ManageBillActivity.tsx

```diff
@@ -0,0 +1,288 @@
+
+export const ManageBillActivity: FC = () => {
+  const { t } = useTranslation();
+
+  const breadcrumbMap = {
+    [t("nav.dashboard")]: "/",
+    [t("nav.billing")]: "/billing",
+    [t("nav.newbill")]: "/bills",
+    [t("nav.managebills")]: "/managebills",
+  };
+  const dispatch = useDispatch();
+
+  const [expanded, setExpanded] = useState<string | false>("form");
+  const [isOpen, setIsOpen] = useState(false);
+  const [fromDate, setFromDate] = useState(
+    new Date().setDate(new Date().getDate() - 6) + ""
+  );
+  const [toDate, setToDate] = useState(new Date().setHours(23) + "");
+  const [patientCode, setPatientCode] = useState(0);
+
+  useEffect(() => {
+    dispatch(searchBills(fromDate, toDate, patientCode));
+  }, [fromDate, toDate, patientCode]);
+
+  const userCredentials = useSelector<IState, TUserCredentials>(
+    (state) => state.main.authentication.data
+  );
+
+  const summary = useSelector<IState, IBillSummary>((state) =>
+    computeBillSummary(
+      state.bills.searchBills.data,
+      fromDate,
+      toDate,
+      userCredentials?.displayName ?? ""
+    )
+  );
+
+  const tabConfig: TTabConfig = [
+    {
+      label: t("bill.allbills"),
+      path: "/allBills",
+      content: (
+        <ManageBillActivityContent
+          title="All Bills"
+          content={
+            <BillTable
+              status="ALL"
+              toDate={toDate}
+              fromDate={fromDate}
+              patientCode={patientCode}
+            />
+          }
+        />
+      ),
+    },
+    {
+      label: t("bill.pending"),
+      path: "/pendingBills",
+      content: (
+        <ManageBillActivityContent
+          title="Pending Bills"
+          content={
+            <BillTable
+              status="PENDING"
+              toDate={toDate}
+              fromDate={fromDate}
+              patientCode={patientCode}
+            />
+          }
+        />
+      ),
+    },
+    {
+      label: t("bill.closed"),
+      path: "/closedBills",
+      content: (
+        <ManageBillActivityContent
+          title="Closed Bills"
+          content={
+            <BillTable
+              status="CLOSE"
+              toDate={toDate}
+              fromDate={fromDate}
+              patientCode={patientCode}
+            />
+          }
+        />
+      ),
+    },
+    {
+      label: t("bill.deleted"),
+      path: "/deletedBills",
+      content: (
+        <ManageBillActivityContent
+          title="Deleted Bills"
+          content={
+            <BillTable
+              status="DELETE"
+              toDate={toDate}
+              fromDate={fromDate}
+              patientCode={patientCode}
+            />
+          }
+        />
+      ),
+    },
+  ];
+
+  const submit = (filter: any) => {
+    setFromDate(filter.fromDate);
+    setToDate(filter.toDate);
+    setPatientCode(filter.patient);
+  };
+
+  const handleOnExpanded = (section: string) => {
+    setExpanded(section === expanded ? "form" : section);
```

Comment:
setExpanded(section === expanded ? **"form"** : section);

should return false instead

Reply:
Right. But I removed the expand feature for the moment, since the user can directly see both sections.